README.md

```diff
@@ -28,6 +28,26 @@ See [**Contributing.md**](./CONTRIBUTING.md) to see how to format your pull requ
 
 ### A
 
+#### **Accord Project**
+
+- **Swag**: T-shirt and Stickers
+- **Requirements**: Create pull requests on [Accord Project's](https://accordproject.org) participating repositories.
+
+  *Participating repositories*
```

Comment:
Please remove this list and instead change it to something like "See the blog post for a list of participating repositories".